lambdas/credentialissuerstart/src/main/java/uk/gov/di/ipv/core/credentialissuer/domain/CriResponse.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.di.ipv.core.credentialissuer.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CriResponse {
+    @JsonProperty private final String id;
+
+    @JsonProperty private final String authorizeUrl;
+
+    @JsonProperty private final String request;
```

Comment:
I don't think we actually need this field anymore. I think the original plan was for this request field to be the "request" property that we send the shared attributes across to the CRI in. But for now we are keeping this as a manual step that core-front will always perform before redirecting to the CRI so it doesn't need to be returned in the CriResponse